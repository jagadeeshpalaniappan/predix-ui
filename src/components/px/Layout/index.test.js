import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import Layout from './';

describe('Layout', () => {
  test('should render', () =>{
    const tree = renderer.create(<Layout>Title</Layout>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
