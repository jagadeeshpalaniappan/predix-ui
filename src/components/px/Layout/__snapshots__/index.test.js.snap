// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout should render 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  list-style: none;
  margin: 0;
  padding: 0;
}

<div
  className="layout c0"
>
  Title
</div>
`;
